Formula/libserdes.rb

```diff
@@ -0,0 +1,40 @@
+class Libserdes < Formula
+  desc "Schema ser/deserializer lib for Avro + Confluent Schema Registry"
+  homepage "https://github.com/confluentinc/libserdes"
+  url "https://github.com/confluentinc/libserdes.git",
+  :tag => "v5.3.1"
```

Comment:
This needs a revision specified as well. Otherwise it could be retagged and we'd never know.

Reply:
Corrected.